lib/vcloud/launcher/vm_orchestrator.rb

```diff
@@ -23,7 +23,7 @@ def customize(vm_config)
 
         preamble = vm_config[:bootstrap] ? generate_preamble(vm_config) : ''
 
-        @vm.configure_guest_customization_section(preamble)
+        @vm.configure_guest_customization_section(preamble) if (!preamble.nil? && !preamble.empty?)
```

Comment:
@joshmyers Is the check for `nil` necessary? Are there any instances under which `preamble` would be `nil`?

If so, the logic here could be simplified to:

```ruby
@vm.configure_guest_customization_section(preamble) unless preamble.nil? || preamble.empty?
```